diff --git a/node_modules/mingo/mingo.js b/node_modules/mingo/mingo.js
index bd1c4de..908cd56 100644
--- a/node_modules/mingo/mingo.js
+++ b/node_modules/mingo/mingo.js
@@ -21,7 +21,7 @@
     return Mingo;
   };
 
-  var nodeEnabled = (undefined !== module && undefined !== require);
+  var nodeEnabled = (undefined !== module && typeof require !== 'undefined');
 
   // Export the Mingo object for Node.js
   if (nodeEnabled) {
